apps/web/tailwind.config.js

```diff
@@ -9,8 +9,8 @@ const {VeniceTheme} = require('./styles/themes')
  */
 module.exports = {
   content: [
-    './{components,pages,screens}/**/*.{ts,tsx}',
-    '../../integrations/*/**/*.tsx',
+    './**/*.tsx',
```

Comment:
I don't think this will affect build (dev & prod) performance. It'll prevent us from forgetting to add it to the list.

I forgot to add this in the previous PR; that was why I had an issue with tailwind (it didn't generate new classes, so only utility classes used else where worked).

wdyt? @tonyxiao

Reply:
Would `    '../../integrations/*/**/*.tsx',` be covered when we use `./**/*.tsx`?

Reply:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/696842/215080446-450ca550-86ed-4ab1-a51d-5908ec8044cc.png">
Do you happen to know why in the official docs they list out the directories rather than simply using a wildcard? Is there something we are not considering?

Reply:
> Would ` '../../integrations/*/**/*.tsx',` be covered when we use `./**/*.tsx`?

It will **not** be since `./**` is local i.e. `apps/web`.

> Do you happen to know why in the official docs they list out the directories rather than simply using a wildcard? Is there something we are not considering?

Just a guess but I believe it's for build perf optimization (so tailwind doesn't accidentally try to work on files we don't intend to). Or maybe just to be explicit. It's just glob patterns to tell Tailwind complier which files to watch & process.

Reply:
Interesting . What's the biggest issue with switching to app directory in the new next?